mlflow/tracing/trace_manager.py

```diff
@@ -84,14 +87,18 @@ def start_detached_span(
 
         try:
             tracer = get_tracer(__name__)
-            if parent_span_id:
-                parent_span = self.get_span_from_id(request_id, parent_span_id)._span
+            if parent_id:
+                parent_span = self.get_span_from_id(request_id, parent_id)._span
```

Comment:
Looks like `request_id` might be `None` at this point based on lines 98 and 99. Do we need to adjust this?

Reply:
That's true, we should add conditional check here and warning (cuz it's invalid parent ID passed by users). Will update and add a test for this case.